pkgs/top-level/stage.nix

```diff
@@ -110,7 +110,7 @@ let
       inherit lib;
       inherit (self) config;
       inherit (self) runtimeShell stdenv stdenvNoCC;
-      inherit (self.pkgsBuildHost) jq shellcheck-minimal;
+      inherit (self.pkgsBuildHost) jq shellcheck-minimal unixtools tmate;
```

Comment:
Because in order to provide `unixtools` and `tmate` to `trivial-builders.nix` so that we can put the `addTmateBreakpoint` function in there. Though as @roberth says, `trivial-builders.nix` may not be the correct location for this to reside.